.travis.yml

```diff
@@ -62,7 +62,7 @@ install:
     fi
   - |
     if ! which kubecfg; then
-      wget -O $GOPATH/bin/kubecfg https://github.com/ksonnet/kubecfg/releases/download/v0.7.1/kubecfg-$(go env GOOS)-$(go env GOARCH)
+      wget -O $GOPATH/bin/kubecfg https://github.com/ksonnet/kubecfg/releases/download/v0.7.2/kubecfg-$(go env GOOS)-$(go env GOARCH)
```

Comment:
Oh wow.

So there never was an osx build of kubecfg 0.7.1 - the osx build failure was the reason 0.7.2 was released.

So what I'm most concerned about here is that we (/I) made the change to use 0.7.1, it passed travis tests, and got merged without ever actually executing this line of code on osx :/

I think this means we should download on every build, or (better) download to a versioned filename (and hence keep the cache) but then always symlink/copy that over to the unversioned executable name.

I'm happy to put in a PR to clean that up tomorrow if you want - just a heads up that it will conflict with this patch.

Reply:
(fixed in #97)